twolevel_dbm.py

```diff
@@ -253,15 +260,37 @@ def secondlevel(inputs, args, secondlevel=False):
                 f"ImageMath 3 output/jacobians/overall/secondlevel_{subjectname}_absolute.nii.gz + output/jacobians/overall/secondlevel_{subjectname}_nlin.nii.gz output/jacobians/overall/secondlevel_{subjectname}_affine.nii.gz")
             pool.uimap(lambda x: run_command(x, args.dry_run), commands)
             commands = list()
+
         jacobians.append(
             f"output/jacobians/overall/secondlevel_{subjectname}_relative.nii.gz")
         jacobians.append(
             f"output/jacobians/overall/secondlevel_{subjectname}_absolute.nii.gz")
         jacobians.append(
             f"output/jacobians/overall/secondlevel_{subjectname}_nlin.nii.gz")
+
     run_command(
         "echo DONE > output/compositewarps/secondlevel/COMPLETE", args.dry_run)
 
+################################################################
+    for i, subject in enumerate(tqdm.tqdm(input_images), start=0):
+        subjectname = pathlib.Path(subject).name.rsplit('.nii')[0]
+        if not secondlevel and args.resample_to_common_space:
+            commands.append(
+                f"antsApplyTransforms -d 3 -i output/jacobians/overall/secondlevel_{subjectname}_relative.nii.gz -t output/secondlevel/template0_common_space_1Warp.nii.gz -t output/secondlevel/template0_common_space_0GenericAffine.mat "
+                f"-r {args.resample_to_common_space} --verbose -o output/jacobians/common_space/secondlevel_{subjectname}_relative.nii.gz")
+            commands.append(
+                f"antsApplyTransforms -d 3 -i output/jacobians/overall/secondlevel_{subjectname}_absolute.nii.gz -t output/secondlevel/template0_common_space_1Warp.nii.gz -t output/secondlevel/template0_common_space_0GenericAffine.mat "
+                f"-r {args.resample_to_common_space} --verbose -o output/jacobians/common_space/secondlevel_{subjectname}_absolute.nii.gz")
+            commands.append(
+                f"antsApplyTransforms -d 3 -i output/jacobians/overall/secondlevel_{subjectname}_nlin.nii.gz -t output/secondlevel/template0_common_space_1Warp.nii.gz -t output/secondlevel/template0_common_space_0GenericAffine.mat "
+                f"-r {args.resample_to_common_space} --verbose -o output/jacobians/common_space/secondlevel_{subjectname}_nlin.nii.gz")
+            pool.uimap(lambda x: run_command(x, args.dry_run), commands)
+            commands = list()
+################################################################
```

Comment:
We need a jacobians.append here for the newly generated files, so they are also processed by the smoothing code at the end, see slightly above.